kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -162,34 +162,36 @@ def __call__(self, x):
 
         kernel = """
         extern "C"
-        __global__ void abs_complex_value(const ${Dtype} * x, ${Dtype}2 * z, int n)
+        __global__ void abs_complex_value(const ${Dtype} * x, ${Dtype} * z, int n)
         {
             int i = blockIdx.x * blockDim.x + threadIdx.x;
         if (i >= n)
             return;
-        z[i] = make_${Dtype}2(normf(2, x + 2*i), 0);
+        z[i] = normf(2, x + 2*i);
```

Comment:
what replaced the casting operation?

Reply:
not casting to complex (we just want real now).

Reply:
I asked a clarification question in the 1D counterpart. I'd like to make sure I understand properly

Reply:
Does my answer help?

Reply:
-ish lol. If it passes the tests, then that is good with me. It seems to.